includes/admin/settings/class-wc-admin-settings-woorule.php

```diff
@@ -146,7 +144,6 @@ public static function order_status_changed($id, $status = '', $new_status = '')
                     }
 
                     $order_data = $order->get_data();
-                    $phone = $order->get_billing_phone();
```

Comment:
It was extra (unneeded) data right? *and phone will be taken from $order_data['billing']['phone']

Reply:
@dmitry-rule yes, it was unused field